packages/cli/src/utils/input.ts

```diff
@@ -238,3 +250,35 @@ export function validateSalt(salt: string, required = false) {
     throw new Error(`Invalid salt ${salt}, must be an uint256 value.`);
   }
 }
+
+export function getPlaceholder(arg: Pick<MethodArg, 'type' | 'components'>): string {
```

Comment:
The compiler isn't catching this because we're not running it in strict mode. It will be a bit of an undertaking to enable it, but I would highly recommend we do.

```suggestion
export function getPlaceholder(arg: Pick<MethodArg, 'type' | 'components'>): string | null {
```

Reply:
Good one. And agree on the strict mode!